### DISCUSSION TITLE
PRS Distribution Statement Definitions (1h)

### DISCUSSION THREAD

**Discussion by richardbirkin** [SuperHyperInstantFutureTime/TrackShift#23] [Work log]
1h @richardbirkin 

Completed the definitions/clarification on all header row items for PRS Distribution Statements [here](https://github.com/SuperHyperInstantFutureTime/Trackshift/wiki/PRS-Distribution-Statement-Column-Definitions)

**Comment by g105b** [comment]
That's going to prove really useful. Great to have it all in one place.

**Reply by richardbirkin** [reply]
Yeah, it was super useful just reminding myself about how the last few columns work to affect the overall total.
